LM IR BLinker.X/main.c

```diff
@@ -56,9 +68,10 @@ void main(void)
 
     PWM3_Initialize();
     TMR0_Initialize();
-    #define IR1 PWM3_LoadDutyValue(51);
-    #define IR0 PWM3_LoadDutyValue(0); 
+    
      // #define CODE    00100010    // Address to transmit out
+    
+    TMR0_SetInterruptHandler(bitTimerInterrupt);

```

Comment:
This is a function that is declared in tmr0.h that allows you to pick your own function as the function that gets called during an interrupt event. You can see here that I've passed the name of the function I want called (a "function pointer") as an "argument" to this function

Reply:
Dope, I was trying to do this with if statements earlier and was failing miserably.